README.md

```diff
@@ -159,6 +159,36 @@ To get sample traces to try run the following command. The traces will be stored
 yarn download:sample-traces
 ```
 
+## Running UI tests
+
+To run the UI tests locally, use the following commands.
+
+Steps for setup that only need to be run once:
+
+```bash
+yarn download:sample-traces
+yarn download:server
+yarn start:server & # or run in a separate shell
```

Comment:
I think this command is to be run whenever one needs to run the tests.

Reply:
I created a new block for starting the servers, hopefully the instructions are clearer now.